packages/ng-schematics/src/ng-new/index.ts

```diff
@@ -48,18 +48,32 @@ export function newProject(options: OptionsSchema): Rule {
 					// app name validation???
 					projLibrary = await prompt.getProjectLibrary(framework);
 
-					const projTemplate = await prompt.getProjectTemplate(projLibrary);
+					let projTemplate;
 
-					// project options:
-					const theme = await prompt.getTheme(projLibrary);
+					if (!options.template) {
+						projTemplate = await prompt.getProjectTemplate(projLibrary);
+					} else {
+						const projects = projLibrary.projectIds;
+						const selectedProj = projects.find((i: string) => i === options.template);
+						if (selectedProj !== undefined) {
+							projTemplate = projLibrary.projects.find((i: { id: any; }) => i.id === selectedProj);
+						} else {
+							projTemplate = await prompt.getProjectTemplate(projLibrary);
```

Comment:
Not sure why go into prompt again, argument was passed and it wasn't found. In this case you usually get an error, no?